### PR TITLE
Server v5.30 Notice

### PR DESCRIPTION
**Pull request by amyblais**
For v5.30 release

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Looks good to me! Thanks!

File: notices.json [hunk 1]
```diff
@@ -17,21 +17,21 @@
...
+      "serverVersion": ["<5.30"],
       "instanceType": "onprem",
-      "displayDate": ">= 2020-11-19T00:00:00Z"
+      "displayDate": ">= 2020-12-19T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "Mattermost server v5.29 is now available offering many bug fixes for increased stability as well as the general availability of a pre-installed incident management application (E20), channel moderation settings (E20), and Mattermost Omnibus. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "Mattermost server v5.30 is now available offering Incident Management out-of-the-box, a new MS Teams Meetings plugin, configuration of new administrator roles from the System Console, and improvements to the Matterpoll plugin. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```
Comment (esethna) [inline notices.json:31]: Propose:
Mattermost server v5.30 is now available offering a new MS Teams Meetings plugin, new administrator roles configurable in the System Console, and improvements to the Matterpoll plugin. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.

_______

> Incident Management out-of-the-box, 
Not really clear how this is different than the text in v5.29 "pre-installed incident management application (E20)"? Propose we just remove it?

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by esethna — DISMISSED** [review]

**Comment by esethna** [comment]
@amyblais this should only be merged after the release is available (or else we are removing the v5.29 notice)

**Comment by amyblais** [comment]
@ogi-m Kind reminder to test this.

**Comment by amyblais** [comment]
@ogi-m v5.30.1 has the fix for the version number.

**Comment by esethna** [comment]
@amyblais I'll be out for holidays now, but please merge this when it's tested and verified

**Review by esethna — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]
Tested and passed
- notice does not show up on cloud servers
- version check working

cc @amyblais

**Pull request merged**